google/ggrc-core
# GGRC-3447 Add default line as attribute query to Unified mapper/ Global Search / Generation popup

# Issue description

Add default line as attribute query to Unified mapper/ Global Search / Generation popup
![image](https://user-images.githubusercontent.com/3773902/34106147-27963320-e408-11e7-8536-52f79fba341c.png)

# Steps to test the changes

1. Open Unified Mapper for any entity
2. Look at the default line as attribute query

**Actual Result:** Default line as attribute query is not displayed in Unified mapper/ Global Search / Generation popup
**Expected Result:** Default line as attribute query should be displayed in Unified mapper/ Global Search / Generation popup 

# Solution description

Set default 'filterItems' value.

# Sanity checklist




# PR Review checklist
